modules/EvseManager/EvseManager.cpp

```diff
@@ -306,5 +583,51 @@ bool EvseManager::reserved_for_different_token(const std::string& token) {
         return true;
     }
 }
+/*
```

Comment:
large commented out code block, is this intentional?

Reply:
will remove